tldr/hybrid_extractor.py

```
@@ -3151,6 +3168,159 @@ def _extract_elixir_function(self, node, source: bytes, is_private: bool) -> Fun
+        tree = self._safe_parse(parser, source, file_path, "zig")
+
+        module_info = ModuleInfo(
+            file_path=str(file_path),
+            language="zig",
+            docstring=None,
+        )
+
+        self._extract_zig_nodes(tree.root_node, source, module_info)
+        return module_info
+
+    def _get_zig_parser(self) -> Any:
```

Ruff (0.14.14)

Dynamically typed expressions (typing.Any) are disallowed in `_get_zig_parser`

(ANN401)